cypress/support/Train.js

```diff
@@ -112,12 +112,12 @@ export function SelectAction(expectedResponse, lastResponse) {
   })
 }
 
-export function SelectEndSessionAction(action_selector, expected_rendered, lastResponse) {
-  scorerModal.ClickEndSessionAction(action_selector, expected_rendered)
+export function SelectEndSessionAction(expectedData, lastResponse) {
```

Comment:
Less than a fan of the signature change. The original had three arguments making the code more flexible as new test cases come online involving more complex actions (e.g. cards, prompts). The payload view displayed in the grid list view may be different than the render() method within the Bot. This point is made more valid as render() is required of API call actions.

Reply:
I remember you said that when you code it the other way. However I disagree.

For the way the UI is now, "EndSession" is a constant, we don't need to pass that around from function to function, and repeat that for every test case that involves EndSession scenarios. Data is the unique item, so we must pass that around.

When we see what that change looks like in the DOM then we can adjust it accordingly, I don't like coding for things that might come.